decidim-core/app/cells/decidim/upload_modal/modal.erb

```diff
@@ -60,8 +60,7 @@
     <%= t("decidim.shared.confirm_modal.cancel") %>
   </button>
   <button type="button" class="button button__sm md:button__lg button__secondary" data-dropzone-save data-dialog-close="<%= modal_id %>" disabled>
-    <%= t("next", scope: "decidim.messaging.conversations.index") %>
-    <%= icon "arrow-right-line", class: "fill-current" %>
+    <%= t("save", scope: "decidim.messaging.conversations.index") %>
```

Comment:
Having conversations in this i18n key is misleading. I see it was already there, but we should fix it while we're working on this. 

```suggestion
    <%= t("save", scope: "decidim.forms.upload.labels") %>
```

(You'd need to change the en.yml file too of course)